### ISSUE TITLE
Links to project media after changing project owner

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#731]
After changing the owner of a project the file path will change to reflect the new owner's user name (e.g. http://www.nottingham.ac.uk/toolkits/USER-FILES/123-user1-Nottingham to http://www.nottingham.ac.uk/toolkits/USER-FILES/123-user2-Nottingham)

This causes problems if the author has manually added links to media within the project by getting the file path from the project properties panel. e.g. if they have uploaded a word document to be downloaded the link they used will be http://www.nottingham.ac.uk/toolkits/USER-FILES/123-user1-Nottingham/media/document.doc which won't be the correct file path after the project user has changed.

Is there anything we can do to get around this? Some code that goes through the xml & fixes file paths like this when the owner is changed?

**Comment by JohnSmith-LT** [comment]
Hi @FayCross 

I've just tried the "Give this project" option in properties with a manually entered link and gave it to another user and looking at the HTML source it has changed the url to that of the new user... so there must be code somewhere to do this... or is this issue fixed?

**UPDATE** - I actually looked at the data.xml file and even though I added the full link given in media tab, the editor rewrites it, before saving, to media/filename.txt - so a few questions:

How to reproduce - ie where are you putting the link that the editor isn't rewriting?
Is the editor now fixing these potential problems but not for legacy projects that are given but never edited?
